scripts/irods/test/test_federation.py

```diff
@@ -2161,3 +2161,221 @@ def test_failure_to_local_from_remote__issue_6497(self):
 	def test_failure_to_local_from_local__issue_6497(self):
 		self.fail_to_overwrite_replica_on_target_resource_test_impl(copy_to='local', copy_from='local')
 
+
+class test_irepl_all_permission_levels__issue__7444_7465(unittest.TestCase):
```

Comment:
`__issue__` -> `__issue_`?

Wouldn't the extra underscore make grepping for `__issue_NNNN` more difficult?
Are there perks to including the extra underscore?

Reply:
That... is a typo, I think. Will remove the extra underscore.

Reply:
Done